src/FileHelper.php

```diff
@@ -412,16 +411,41 @@ public static function copyFile(string $source, string $destination, array $opti
                 chmod($destination, $fileMode);
             }
 
-            if ($afterCopy) {
-                $afterCopy($source, $destination);
-            }
+            self::processCallback($afterCopy, $source, $destination);
 
             return true;
         }
 
         return false;
     }
 
+    /**
+     * @param mixed $callback
+     * @param array $arguments
+     *
+     * @throws InvalidArgumentException
+     *
+     * @return mixed
+     */
+    private static function processCallback($callback, ...$arguments)
+    {
+        if ($callback === null) {
+            return;
+        }
+
+        if ($callback instanceof Closure) {
+            return $callback(...$arguments);
+        }
```

Comment:
```suggestion
        if ($callback instanceof Closure) {
            return $callback(...$arguments);
        }
```

Unnecessary code. Next `call_user_func_array()` can run `Closure`.